### DISCUSSION TITLE
How to use DragAndDrop function in mvvm mode?

### DISCUSSION THREAD

**Discussion by howzew** [AvaloniaUI/Avalonia#14184] [Q&A] [answered]
I looked at this example [DragAndDropPage](https://github.com/AvaloniaUI/Avalonia/blob/master/samples/ControlCatalog/Pages/DragAndDropPage.xaml.cs) If I want to get the dragged file information in the ViewModel, how should I do it? No examples seem to be found

**Comment by timunie** [comment, marked as answer]
you will need to wire a Behavior or use ViewInteraction. 
- https://github.com/AvaloniaUI/Avalonia.Samples?tab=readme-ov-file#view-interaction-samples
- https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors
